### ISSUE TITLE
Issues logging in

### ISSUE THREAD

**Issue by bigbadmonster17** [turt2live/matrix-trello-bot#47]
I saw this bot and thought it'd be useful for my software project management class. I attempted logging into the bot hosted by t2bot.io but i was met with this error.

Invalid return_url. Wildcard ("*") allowed origins are no longer supported. Contact the developer. See 
https://community.developer.atlassian.com/t/pure-wildcard-allowed-origins-on-app-keys-will-no-longer-be-recognized-and-will-be-fully-deprecated/52093/

After a quick browse through it looks like this is something I am not able to resolve this on my own. Is there anything i can do to get this bot to run? Will this work if I self host it?

Thanks

**Comment by DoStraTech** [comment]
same here.
Any help or a way to self host would be apreciated. 
Thanks!

**Comment by paulinsche** [comment]
As long as this is not fixed, new users are simply excluded.

**Comment by larrasket** [comment]
same issue

**Comment by mjtch-rll** [comment]
If this is self-hosted, does the issue resolve itself? Or are new users simply unable to use this bot?

**Comment by larrasket** [comment]
@mjtch-rll I don't think it does, you will still have to obtain trello API key AFAIK. Also I do not think that the issue is related to the host.

**Comment by JeremyJalpha** [comment]
also getting the same.
:: Invalid return_url. Wildcard ("*") allowed origins are no longer supported. Contact the developer. See https://community.developer.atlassian.com/t/52093
